terra-jupyter-aou/gitignore_global

```diff
@@ -0,0 +1,17 @@
+# By default, all files should be ignored by git.
+# We want to be sure to exclude files containing data such as CSVs and images such as PNGs.
+*.*
+# Now, allow the file types that we do want to track via source control.
+!*.ipynb
+!*.py
+!*.r
+!*.R
+!*.wdl
+!*.sh
+# Allow documentation files.
+!*.md
+!*.rst
+!LICENSE*
+# Also allow some particular source control metadata files.
+!.git*
```

Comment:
I went ahead and removed it altogether since git gives a pretty good message when you try to add something that is ignored:
```
jupyter-user@58f37502814d:/tmp/workbench-snippets$ git add .github/workflows/should_be_visible.yml 
The following paths are ignored by one of your .gitignore files:
.github/workflows/should_be_visible.yml
Use -f if you really want to add them.
```

If people want to add those sorts of files, they are likely comfortable using `git -f` to add them.